_plugins/rosindex_generator.rb

```diff
@@ -831,225 +831,227 @@ def generate(site)
       $all_distros.reverse_each do |distro|
 
         puts "processing rosdistro: "+distro
+        # TODO(tfoote) For loop here
```

Comment:
@tfoote I don't fully grasp this TODO.

Reply:
Sorry this todo was resolved I forgot to remove it